### DISCUSSION TITLE
How do I inline HTML inside another HTML file?

### DISCUSSION THREAD

**Discussion by icanfathom** [parcel-bundler/parcel#8483] [Q&A]
I want to inline one HTML file inside another. This seems like something Parcel could do using the [transformer-inline](https://www.npmjs.com/package/@parcel/transformer-inline) plugin, but I can't figure out how to implement it.

Say I have `footer.html`:
```
<footer>
   © Copyright 2022
</footer>
```

And `index.html`:
```
<body>
  bundle-inline:./footer.html  
</body>
```

My questions:

1.  Is HTML inlining possible?
2. What's the syntax?

More general to bundle inlining (these weren't clear to me from the [docs](https://parceljs.org/features/bundle-inlining/)):

3. Does Parcel just do a global string search for any named transformers ('bundle-inline') and replace them at build time? Or is there a character/word it looks for ('@import')?
4. Are transformers such as [transformer-inline](https://www.npmjs.com/package/@parcel/transformer-inline) automatically included once you call them (similar to how languages and plugins are auto installed)?
5. Do you have to include non-default transformers in the .parcelrc for auto-install?

**Comment by mischnic** [comment]
This might be possible with some plugin, but not with the existing ones.

One solution is using `posthtml-include` or `posthtml-import`: https://parceljs.org/languages/html/#posthtml

**Reply by icanfathom** [reply]
Cool, I'll look into this.

What are the limits to the built-in plugins? This wasn't clear to me from the docs or examples.
